cg/constants/delivery.py

```diff
@@ -78,10 +78,11 @@
 
 CLINICAL_DELIVERY_TAGS: list[set[str]] = [{HermesFileTag.CLINICAL_DELIVERY}]
 
-FASTQ_ANALYSIS_CASE_TAGS: list[set[str]] = []
+RAW_DATA_ANALYSIS_CASE_TAGS: list[set[str]] = []
 
-FASTQ_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
+RAW_DATA_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
     {"fastq"},
+    {"bam"},
```

Comment:
hmm not sure if I like how this could end up ultimately... it would start pulling BAM files for fastq-analysis if these are there... 

Maybe we should re think what raw data really means.